FH.framework/Versions/A/Headers/FHSyncDataset.h

```diff
@@ -0,0 +1,71 @@
+//
+//  FHSyncDataset.h
+//  fh-ios-sdk
+//
+//  Copyright (c) 2012-2015 FeedHenry. All rights reserved.
+//
```

Comment:
@edewit Those copyright below to the fh-ios-sdk itselt. let me do a PR on it and release a 3.0.0-beta2